contracts/RLPReader.sol

```diff
@@ -4,7 +4,7 @@
  * @author Hamdi Allam hamdi.allam97@gmail.com
  * Please reach out with any questions or concerns
  */
-pragma solidity >=0.5.10 <=0.8.18;
+pragma solidity >=0.5.0 <0.9.0;
```

Comment:
Should we specify ">=0.5.10" instead of ">=0.5.0"?

Reply:
Any particular reason we want to bump the lower bound?